clcache.py

```diff
@@ -792,9 +792,9 @@ def _parseOptionsAndFiles(cmdline):
             'Ob', 'Gs', 'Fa', 'Fd', 'Fm',
             'Fp', 'FR', 'doc', 'FA', 'Fe',
             'Fo', 'Fr', 'AI', 'FI', 'FU',
-            'D', 'U', 'I', 'Zp', 'vm',
+            'D', 'U', 'I', 'Zp',
             'MP', 'Tc', 'V', 'wd', 'wo',
-            'W', 'Yc', 'Yl', 'Tp', 'we',
+            'Yc', 'Yl', 'Tp', 'we',
```

Comment:
I don't think this is correct for Visual Studio 2013. The [documentation](https://msdn.microsoft.com/en-us/library/thxezb7y(v=vs.120).aspx) explains that `/W` takes a numeric argument, e.g. both `/W4` as well as `/W 4` are valid.